net/keepalived/Makefile

```diff
@@ -32,12 +32,12 @@ define Package/keepalived
   DEPENDS:=+libnl-genl +libopenssl
 endef
 
-define Package/keepalived/description
- Failover and monitoring daemon for Linux Virtual Server (LVS) clusters.
+define Package/keepalived/config
+  source "$(SOURCE)/Config.in"
 endef
 
-define Package/keepalived/conffiles
-/etc/keepalived/keepalived.conf
```

Comment:
well, then let's agree here on a default:
do we keep keepalived's keepalived.conf or 
use the new UCI config as default ?